docs/src/main/paradox/typed/index.md

```diff
@@ -10,6 +10,7 @@ project.description: Using Apache Pekko to build reliable multi-core application
 * [actors](actors.md)
 * [actor-lifecycle](actor-lifecycle.md)
 * [interaction patterns](interaction-patterns.md)
+* [handling responses in Scala 3](handling-actor-responses-scala3.md)
```

Comment:
`handling-actor-responses-with-scala3.md`

Reply:
Good suggestion! Will change the text in the link too.